.env.production

```
@@ -0,0 +1,30 @@
+# Created by Vercel CLI
+ALLO_GRAPHQL_ENDPOINT="https://grants-stack-indexer-v2.gitcoin.co/graphql"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)